unit_tests/dts-boot

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ $TEST_DES == "y" ] && [ ! -f cloud-pass ]; then
+  rm cloud-pass
+  if [ $BOARD_VENDOR == "Notebook" ]; then 
+    # For NVC testing
+    echo "SHY8Lfteq2bCLGD" >> cloud-pass
```

Comment:
This script removes and initializes the file over and over. SOmeone may accidentally add this file to commits with constant changes, which is undesired.

The directory can be shared, no problem. I have just put the v1.7.2 public binaries there, so no problem. The only problem is that I have to remove my MSI credentials, which I forgot...

Reply:
I think I will do the same and put v1.1.1 MSI binary as it would be v1.1.2 and v1.1.3....

Reply:
@miczyg1 I can see MSi credentials are removed, but do you plan to address this as well: `I think I will do the same in new share and put v1.1.1 MSI binary as it would be v1.1.2 and v1.1.3....` ?

Reply:
Yes, I intend to.

Reply:
https://github.com/Dasharo/meta-dts/pull/95/commits/3255e04162e7b4915ab18fd3b0c7f59a26c08b43

Reply:
Creds added so I will resolve this thread